src/components/Select/MultiSelect.scss

```diff
@@ -0,0 +1,66 @@
+@import '~om';
+
+.MultiSelect {
+  $self: &;
+
+  min-height: 2.5625rem;
+  position: relative;
+  display: flex;
+  width: 100%;
+
+  &__search {
+    @include input-styles();
+    display: flex;
+
+    &--error {
+      border: 1px solid color(code-red);
+    }
+  }
+
+  &__selected-items {
+    display: flex;
+    gap: 0.3rem;
+    max-width: 80%;
+    overflow: scroll;
```

Comment:
```suggestion
    overflow: auto;
```
Showing the scrollbar always makes it look weird:
![image](https://user-images.githubusercontent.com/18576413/122894101-94f6a000-d34f-11eb-9978-1c3fea9324de.png)

Reply:
I mean to really make it scrollable on the x axis but I really only want to show that scrollbar if the width is higher than the max width of the container

Reply:
Right, `overflow: auto` does exactly that I think.